src/controllers/login.js

```diff
@@ -0,0 +1,44 @@
+import UserService from '../services/UserService';
+import { isPasswordTrue } from '../utils/paaswordDecrypt';
+import generateToken from '../utils/tokenHandler';
+
+/**
+   * function findAll() returns all users in db
+   * @param {object} req it request
+   * @param {object} res it response
+   * @returns {object} send reponse to the user
+   */
+const getUser = async (req, res) => {
```

Comment:
you might also want to rename this arrow function to something more descriptive like `userLogin`

Reply:
done